graphs/views.py

```diff
@@ -297,6 +297,18 @@ def _graphs_page(request, view_type):
 
     return render(request, 'graphs/graphs.html', context)
 
```

Comment:
Add python documentation for this method. https://www.python.org/dev/peps/pep-0257/